Datahub.Core/Components/MudFormTemplate.razor

```diff
@@ -215,21 +218,24 @@ else
 
         if (Context.ChangeTracker.HasChanges())
         {
-            var messages = new List<string>() { "Would you like to save the current record?" };
-            await UI.ToggleModal(@<UserChoiceModal HeaderText="Save Data" ChoiceTextList=@messages OnDecision="(a) => OnConfirmSave(a, selectedRecord)" ConfirmText="Yes" CancelText="No" />);
+            bool? result = await DialogService.ShowMessageBox(
+                "Warning",
+                "Would you like to save the current record?",
```

Comment:
Localize these texts later.

Reply:
Thanks. I will localize in phase 2 of the project :)